content/courses/connecting-to-offchain-data/oracles.md

```diff
@@ -473,21 +476,21 @@ Once you have found a feed that fits your needs, you still need to make sure
 you're using the feed appropriately. For example, you should still implement
 necessary security checks on the account passed into your instruction. Any
 account can be passed into your program's instructions, so you should verify
-it’s the account you expect it to be.
+it's the account you expect it to be.
 
 In Anchor, if you deserialize the account to the `AggregatorAccountData` type
-from the `switchboard_v2` crate, Anchor checks that the account is owned by the
-Switchboard program. If your program expects that only a specific data feed will
-be passed in the instruction, then you can also verify that the public key of
-the account passed in matches what it should be. One way to do this is to hard
-code the address in the program somewhere and use account constraints to verify
-the address passed in matches what is expected.
+from the `switchboard_solana` crate, Anchor checks that the account is owned by
+the Switchboard program. If your program expects that only a specific data feed
+will be passed in the instruction, then you can also verify that the public key
+of the account passed in matches what it should be. One way to do this is to
+hard code the address in the program somewhere and use account constraints to
+verify the address passed in matches what is expected.
 
 ```rust
 use {
   anchor_lang::prelude::*,
   solana_program::{pubkey, pubkey::Pubkey},
-	switchboard_v2::{AggregatorAccountData},
+  switchboard_solana::{AggregatorAccountData},
```

Comment:
Format this (just paste into deleteme.rs in vscode, save it so it formats, then paste back)

Reply:
Yes formatted with deleteme file.